guide/src/installation.md

```diff
@@ -1,16 +1,69 @@
 # Install the relayer
 
-In order to install and run Hermes, please follow the steps below:
+There are two main approaches for obtaining Hermes:
 
-> NOTE: This assumes you have installed all the [pre-requisites](./pre_requisites.md) on your machine.
+1. Installation:
+   1. If you are running on a Unix machine (Linux/MacOS), then the simplest
+      option is to [download the latest binary](#Install-by-downloading).
+   2. Alternatively, install via
+      [Cargo installation instructions](#install-via-cargo). 
+
+[comment]: <> (   Regardless of approach you choose, also follow the post-installation)
+
+[comment]: <> (   instructions therein.)
+
+2. Alternatively, [build Hermes directly from source](#build-from-source).
+
+
+## Install by downloading
+
+Simply head to the GitHub [Releases][releases] page download the latest version
+of Hermes binary matching your platform:
+- `hermes-v0.1.1-x86_64-apple-darwin.tar.gz` (or .zip) for MacOS,  
+- `hermes-v0.1.1-x86_64-unknown-linux-gnu.tar.gz` (or .zip) for Linux.
+
+The step-by-step instruction below should carry you through the whole process:
+ 
+1. Make the directory where we'll place the binary:
+   ```shell
+   mkdir $HOME/.hermes/
```

Comment:
```suggestion
   mkdir -p $HOME/.hermes/bin
```

Reply:
Better to put the binary in its own directory so that we don't pollute the PATH with non-executables later on.